stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
Or maybe even `Some possible reasons for returning [None]:`? Why all the bureaucracy?